PortalSwift/Sources/PortalSwift/Storage/GDrive/GDriveClient.swift

```diff
@@ -0,0 +1,305 @@
+//
+//  GDriveClient.swift
+//  PortalSwift
+//
+//  Created by Blake Williams on 2/7/23.
+//
+
+import Foundation
+import GoogleSignIn
+
+public enum GDriveClientError: Error {
```

Comment:
Here we have the errors outside of the class but in icloud we have the errors inside the class. Is that intended?

Reply:
I didn't refactor that at all, I just think they were written differently to begin with. I'm fine with settling on a convention, though.